src/components/PositionedOverlay/PositionedOverlay.tsx

```diff
@@ -118,20 +119,30 @@ export class PositionedOverlay extends React.PureComponent<
   }
 
   render() {
-    const {left, top, zIndex, width} = this.state;
-    const {render, fixed, classNames: propClassNames} = this.props;
```

Comment:
Removing `propClassNames` and not passing to `classNames` below broke popover CSS transitions. I opened an issue for it #2310 2310

|Before|After|
|---|---|
|![before](https://user-images.githubusercontent.com/344839/66979023-eae65100-f060-11e9-890b-8b19a0630040.gif)|![after](https://user-images.githubusercontent.com/344839/66979029-f20d5f00-f060-11e9-896a-f3b7338f6cbf.gif)|